unified/unified-ats/package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "@openint/unified-ats",
   "dependencies": {
+    "@openint/cdk": "workspace:*",
```

Comment:
Technically should only need `vdk` not `cdk` as `vdk` stands for `vertical development kit`

Reply:
It's just for the types in L3. This is also used by the bankingLink

import type {AnyEntityPayload, Id, Link} from '@openint/cdk'